tests/test_cargo_install.rs

```diff
@@ -83,6 +83,45 @@ test!(simple {
     assert_that(cargo_home(), is_not(has_installed_exe("foo")));
 });
 
+test!(multiple_pkgs {
+    pkg("foo", "0.0.1");
+    pkg("bar", "0.0.1");
+
+    assert_that(cargo_process("install").arg("foo").arg("bar"),
+                execs().with_status(0).with_stdout(&format!("\
+{updating} registry `[..]`
+{downloading} foo v0.0.1 (registry file://[..])
+{compiling} foo v0.0.1 (registry file://[..])
+{installing} {home}[..]bin[..]foo[..]
+{updating} registry `[..]`
```

Comment:
Hm this seems kinda unfortunate (updating the registry twice). Perhaps the "loop over all packages" could be pushed a little deeper to prevent this from happening?